dev-aws/kafka-shared-msk/account-identity/legacy_account.tf

```diff
@@ -134,6 +134,13 @@ module "cbc_account_events_relay_v2" {
   cert_common_name = "cbc/cbc-account-events-relay-v2"
```

Comment:
and this match https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/dev-aws/kafka-shared-msk/cbc/cbc.tf#L1300

is that an issue?